tools/tests/test_lint.py

```
@@ -1144,3 +1456,313 @@ def test_non_mapping_dispositions_and_staffing_are_findings(tmp_path):
+    ("a question mark followed by a space", "? a description", True),
+    ("a bare indicator", "-", True),
+    # Not covered by "an inline comment", which pins the ` #` check. A
+    # leading `#` loads as null with the vendor silent under both endings,
+    # so this guard is the only thing catching it -- and the two-set split
+    # is what makes this branch one a mutation can move.
+    ("a leading hash", "#leading hash", True),
+]
+
+
+@pytest.mark.parametrize(
+    "value,fires",
```

Ruff (0.16.2)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)